packages/ui/react-ui-form/src/components/deprecated/FormInput.tsx

```diff
@@ -27,7 +41,9 @@ export const DeprecatedFormInput = ({
       <Input.Root>
         <div role='none' className='flex flex-col w-full'>
           {/* TODO(burdon): Consistent height for controls (e.g., Select, Textbox, and Checkbox are all different). */}
-          <Input.Label classNames='flex min-h-[40px] items-center'>{label}</Input.Label>
+          <Input.Label classNames='flex min-h-[40px] items-center' style={{ fontSize: 'unset' }}>
```

Comment:
```suggestion
          <Input.Label classNames='flex min-bs-[--rail-action] items-center' style={{ fontSize: 'unset' }}>
```
We should prefer `rem` except for hairline borders, and ideally use tokens when available. `--rail-action` is the equivalent of 40px on desktop.